src/types.ts

```diff
@@ -419,4 +419,10 @@ export type IntercomProviderProps = {
    * Format https://${INTERCOM_APP_ID}.intercom-messenger.com
    */
   apiBase?: string;
+  /**
+   * Indicates if the intercom initialization should be delayed
```

Comment:
Mention that this is in ms

Reply:
Done

Reply:
Done in https://github.com/devrnt/react-use-intercom/pull/236/commits/2b733463f2a223babaf1e262063b291c1882bf6a